src/main/java/mate/academy/Main.java

```diff
@@ -19,7 +18,6 @@ public static void main(String[] args) {
                     .thenAccept(userData -> System.out.println("Processed: " + userData));
         }
 
-        // Wait for all futures to complete
         CompletableFuture.allOf(futures).join();
         executor.shutdown();
```

Comment:
Before shutting down the executor, it's recommended to first call shutdown() to stop accepting new tasks and then call awaitTermination() with a timeout to ensure all tasks have completed or the timeout has elapsed. This helps to gracefully shut down the executor service.